downstream/modules/eda/ref-eda-logging-samples.adoc

```diff
@@ -0,0 +1,77 @@
+[id="eda-logging-samples"]
+
+= Logging samples
+
+When the following APIs are called for each operation, you see the following audit logs:
+
+.Rulebook activation
+
+----
+1. Create
+    1. 2024-08-15 14:13:20,384 aap_eda.api.views.activation INFO   Action: Create / ResourceType: RulebookActivation / ResourceName: quick_start_project / ResourceID: 53 / Organization: Default
+2. Read
+    1. 2024-08-15 14:21:26,844 aap_eda.api.views.activation INFO   Action: Read / ResourceType: RulebookActivation / ResourceName: quick_start_activation / ResourceID: 1 / Organization: Default
+3. Update
+    1. There is no capability to update rulebook activation, must be deleted/recreated
```

Comment:
```suggestion
```
This can be removed I think, we shouldnt note operations which are unsupported in examples I think.
If this is something we do elsewhere, we should keep though.